Packs/CrowdStrikeMalquery/Integrations/CrowdStrikeMalquery/CrowdStrikeMalquery.py

```diff
@@ -0,0 +1,425 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+VENDOR_NAME = 'CrowdStrike Malquery'
```

Comment:
should be exactly as the integration ID (or add the space to the id)
```suggestion
VENDOR_NAME = 'CrowdStrikeMalquery'
```